### PR TITLE
Add --with-privileges options

### PR DESCRIPTION
**Pull request by Swizz**
Hi !

We just found out it was needed in some cases to get the privileges migrations.
Populating a new env using a migration without any privileges setted up do not suit enough our needs.

What do you think ?

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 # Change Log
 
+## v0.2.1 (unreleased)
+
+* Add `--wuth-privileges` option to `diff` and `check` commands.
```
Comment (bikeshedder) [inline CHANGELOG.md]: Please fix that typo.

**Review by bikeshedder — COMMENTED** [review]
I do wonder if that option should also (or rather be) be part of the configuration file?

What's the actual use case here? tbh. I've never included privileges in my migrations.

File: CHANGELOG.md (hunk 1 above)
Comment (Swizz) [reply CHANGELOG.md]:
```suggestion
* Add `--with-privileges` option to `diff` and `check` commands.
```

**Comment by Swizz** [comment]
Here, we keep track of our entire database into a git repository. We create the schema by hand, then we use Tusker to generate the migration file to update on CI job the database according to the change made in the repository.

We also manage dynamic environments. Populating the new empty DB by this mean. 

Without keeping track of the privileges in our repository, the new schema will not have any privileges set, and the job in need of the database will not be granted to use the tables as intended.

**Comment by bikeshedder** [comment]
Interesting use case. Never thought about using a schema migration tool that way. :+1:

**Comment by Swizz** [comment]
We are trying to achieve the whole insfrastructure as code thing.

Dont forget to publish :smile:

**Comment by bikeshedder** [comment]
Published. :+1: 

https://pypi.org/project/tusker/0.2.1/

**Pull request merged**